tests/TestCase.php

```diff
@@ -28,4 +28,18 @@ public function createApplication()
 
         return $app;
     }
+
+    /**
+     * Disable valid_phone validator.
+     * @return $this
+     */
+    protected function withoutPhoneValidation()
+    {
+        Validator::extend('valid_phone', function ($attribute, $value, $parameters, $validator) {
+            // Skip validation because we can't validate a phone number on test creds
+            return true;
+        });
+
+        return $this;
```

Comment:
Looks like we're good here, the setUp refreshes the container before each test and it resets everything. Would you rather have a trait and extent setUpTraits the same as the illuminate test cast?